src/mam4xx/water_uptake.hpp

```diff
@@ -301,6 +306,129 @@ void modal_aero_water_uptake_rh_clearair(const Real temperature,
   rh = haero::max(rh, 0.0);
 }
 
+KOKKOS_INLINE_FUNCTION
+void get_e3sm_parameters(
+    int nspec_amode[AeroConfig::num_modes()],
+    int lspectype_amode[maxd_aspectype][AeroConfig::num_modes()],
+    Real specdens_amode[maxd_aspectype], Real spechygro[maxd_aspectype]) {
+
+  const int ntot_amode = AeroConfig::num_modes();
+  int nspec_amode_temp[ntot_amode] = {7, 4, 7, 3};
+
+  for (int i = 0; i < ntot_amode; ++i) {
+    nspec_amode[i] = nspec_amode_temp[i];
+  }
+
+  Real specdens_amode_temp[maxd_aspectype] = {
+      0.1770000000E+04, 0.1797693135 + 309, 0.1797693135 + 309,
```

Comment:
The second and third values here look fishy to me (and also those on line 329), but they're the same, so it must be intentional(?).

Reply:
Yeah, they looked fishy to me too but are values output directly by e3sm, see[ here](https://github.com/eagles-project/e3sm_mam4_refactor/blob/c658ed322a9cc2abb9fe6dce85d45cebe872ac00/components/eam/src/chemistry/yaml/cam_ndrop/dropmixnuc_input_ts_1400.yaml).

@odiazib